jobs/jobs.go

```diff
@@ -314,10 +314,10 @@ func (job *Job) cleanup(ctx context.Context, cancel context.CancelFunc) {
 		}
 	}
 	cancel()
-	job.Unsubscribe(job.Bus) // deregister from events
+	job.exec.CloseLogs()
```

Comment:
Hm, that really doesn't seem to do anything does it? If `c == nil` then we'd get a NPE when we called `CloseLogs` anyways. I wonder if it found its way into the function during a refactor. Will fix.